binr/r2pm/d/r2pipe-go

```diff
@@ -0,0 +1,21 @@
+R2PM_BEGIN
+
+R2PM_GIT "https://github.com/radare/r2pipe.go"
+R2PM_DESC "[syspkg-r2pipe] r2pipe API for Go"
+
+R2PM_INSTALL() {
+	go get github.com/radare/radare2
```

Comment:
Is this really needed? Why are the bindings installed instead of only r2pipe-go?